SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -15,6 +15,105 @@ DECLARE_LOG_CATEGORY_EXTERN(LogSpatialGDKSettings, Log, All);
 
 class ASpatialDebugger;
 
+USTRUCT()
```

Comment:
not sure if this should be in the Runtime settings. Afaik these settings are not used by workers during runtime, so I'm a bit hesitant to move them there.

after thinking about it: I think I prefer having the worker launch config in the editor settings. we can specify the worker types here, though I think even those should be in the editor settings and then a dropdown to select the correct worker in the runtime settings.

but might be good to get a second opinion on this.